src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImageFunctionalTest.groovy

```diff
@@ -54,12 +62,130 @@ class DockerSaveImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         file(CONTROL_SAVED_IMAGE).size() > file(COMPRESSED_IMAGE_FILE).size()
     }
 
-    static String pullImageTask() {
+    @Unroll
+    def "only saves specific Docker image [image: #image]"() {
+        buildFile << pullImageTask('pullImage', IMAGE_3_4)
+        buildFile << pullImageTask('pullImage2', IMAGE_3_3)
+        buildFile << """
+            import com.bmuschko.gradle.docker.tasks.image.DockerSaveImage
+
+            task saveImage(type: DockerSaveImage) {
+                dependsOn tasks.withType(DockerPullImage)
+                image = "${image}"
+                destFile = file("${IMAGE_FILE}")
+            }
+        """
+
+        when:
+            build('saveImage')
```

Comment:
Can you align the indentation with `when`/`then`/`where` across the board? We don't add a tab to the next line in this code base.

Reply:
Whoops, sorry.
Usually I adapt to the surrounding style.
I'm just too used to it and my formatter settings do it automatically as it imho majorly improves readability. :-)